.travis.yml

```diff
@@ -1,18 +1,19 @@
+bundler_args: --without development --retry=3 --jobs=3
+cache: bundler
 language: ruby
 rvm:
+  - rbx-2
+  - jruby
+  - jruby-head
+  - ruby-head
   - 1.9.2
   - 1.9.3
   - 2.0.0
   - 2.1
   - 2.2
-  - jruby
-  - jruby-head
-  - rbx-2
-  - ruby-head
 matrix:
   allow_failures:
     - rvm: ruby-head
     - rvm: jruby-head
-before_install:
- - sudo apt-get update
- - sudo apt-get install idn
```

Comment:
Good point, but might be better to do a branching check to see if it's installed or not and if not, then install it, just so we don't have to go back and fix if it becomes a problem.